examples/react-webpack-worker-loader/package.json

```diff
@@ -18,7 +18,7 @@
     "file-loader": "^5.1.0",
     "html-webpack-plugin": "^3.2.0",
     "monaco-editor": "^0.20.0",
-    "monaco-yaml": "^2.4.0",
+    "monaco-yaml": "file:../..",
```

Comment:
Is this means load monaco-yaml from dist? or the main code?

Reply:
From the main code. The 2.4.1 update's structure now allows us to do this.